cli/cmd/auth.go

```diff
@@ -32,6 +36,26 @@ More precisely, the Keptn CLI stores the endpoint and API token using *pass* in
 	Example:      `keptn auth --endpoint=https://api.keptn.MY.DOMAIN.COM --api-token=abcd-0123-wxyz-7890`,
 	SilenceUsage: true,
 	RunE: func(cmd *cobra.Command, args []string) error {
+
+		var err error
+		if *exportConfig {
+			exportEndPoint, exportAPIToken, err = credentialmanager.NewCredentialManager().GetCreds()
+			if err != nil {
+				return err
+			}
+			fmt.Println("Endpoint = ", exportEndPoint.String())
+			fmt.Println("API Token = ", exportAPIToken)
```

Comment:
Please use colons instead of equals, and adapt the output as follows:
```
			fmt.Println("Endpoint: ", exportEndPoint.String())
			fmt.Println("API Token: ", exportAPIToken)
```